src/microns_to_nwb/tools/behavior/behavior.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from phase3 import nda
+from pynwb import TimeSeries
+from pynwb.behavior import PupilTracking, SpatialSeries, EyeTracking
+
+
+def add_eye_tracking(scan_key, nwb):
+    pupil_min_r, pupil_maj_r, pupil_x, pupil_y, timestamps = (nda.RawManualPupil() & scan_key).fetch1(
+        "pupil_min_r", "pupil_maj_r", "pupil_x", "pupil_y", "pupil_times"
+    )
+
+    pupil_min_r = TimeSeries(
+        name="pupil_min_r",
+        description="minor axis of pupil tracking ellipse",
+        data=H5DataIO(pupil_min_r, compression=True),
+        timestamps=H5DataIO(timestamps, compression=True),
+        unit="n.a.",
```

Comment:
Anything available for this?

Reply:
From https://bossdb-open-data.s3.amazonaws.com/iarpa_microns/minnie/functional_data/functional_data_technical_documentation_v3.pdf, apparently 'pixels'?

Reply:
Yes but if I remember correctly from one of our discussion with @bendichter 'pixels' is not an SI unit and therefore should be "n.a." ?

Reply:
Now I'm a little confused - the values in this data (from the example file) are `float64` and scale continuously over a range of roughly 20-60, that doesn't sound like units 'pixels' either? Are they already scaled by those `um_width`/`um_height` factors?

Reply:
Do you suggest they sound too big? This is what I found [in the article](https://www.biorxiv.org/content/10.1101/2021.07.28.454025v2.full.pdf) about pupil diameter tracking: 

> The position of the mirror and camera were manually calibrated per session and focused on the pupil. 
> Field of view was manually cropped for each session (ranging from 828 x 1217 pixels to 1080 x 1920 pixels at ~20 Hz), such that the field of view contained the superior, frontal, and inferior portions of the facial silhouette as well as the left eye in its entirety. Frame times were time stamped in the behavioral clock for alignment to the stimulus and scan frame times.

Reply:
Using https://www.researchgate.net/figure/Pupils-dilate-when-mice-move-a-Simultaneous-recordings-of-pupil-size-and-movement-of_fig13_330367758 as a refence for typical magnitude of this quantity, I guess it's some kind of continuous mapping from the discrete pixel space to a continuous estimate in units of those pixels, which is what threw me a bit. It's too small to be scaled to micrometers

I tried estimating what this range would be like if it were scaled by the effective size of the pixels - I tried inferring the value from the grid spacing and field of view stuff I commented on below but still not getting realistic magnitude based on that.

Can you report here, precisely, the values of `um_width`/`um_height`, and `px_widith`/`px_height`?

Reply:
Hmm pixels is indeed a rather inconvenient type of unit. Fig 6 of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8588114/ plots using 'their' pixels.

Other literature perhaps smartly corrects for all this by normalizing against the session average (Fig 1 of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7494826/), making it a dimensionless quantity.

Still, for this, to maximize an individuals ability to reuse the data, this all needs to be indicated somehow somewhere. At the very least in the `description`, I'd include the detail that the series values are estimated in the relative pixel units. Despite them not being SI units, the abbreviation `px` might still be useful to set as the `unit` since they are, in truth, the units of the series whereas specifying `n.a.` (as distinct from `a.u.` if this was normalized) makes it seem like the units don't matter, but they definitely do.